CHANGELOG.develop

```diff
@@ -977,6 +977,7 @@ Other:
   - Control auto-newline using the =c-c++-enable-auto-newline= variable
     (thanks to Fangrui Song and Codruț Constantin Gușoi)
   - Added =org-babel= support (thanks to Michael Rohleder)
+  - Added debug support via =dap= layer (Firefox and Google Chrome)
```

Comment:
Just noticed that this line is in the wrong group, should be in the javascript section.